packages/web-components/package.json

```diff
@@ -9,53 +9,42 @@
     "type": "git",
     "url": "git+https://github.com/reapit/foundations.git"
   },
-  "license": "MIT",
-  "files": [
-    "pidist-npm/**/*"
-  ],
-  "main": "dist-npm/index.js",
-  "types": "dist-npm/index.d.ts",
-  "typings": "dist-npm/index.d.ts",
-  "scripts": {
-    "build:cdn": "rimraf dist && rimraf dist-npm && webpack --color --mode production --config ../../scripts/webpack/webpack.cdn.js",
-    "build:npm": "rimraf dist-npm && webpack --color --mode production --config ../../scripts/webpack/webpack.basic.js && yarn build:tsc",
-    "build:prod-temporary-disable": "echo 'disabling for now yarn build:npm && yarn build:cdn'",
-    "build:tsc": "tsc -p './tsconfig.json'",
-    "check-version": "node ./scripts/check-version.js",
-    "release:dev-temporary-disable": "echo 'disabling for now node ../../scripts/release/release-dev.js web-components reapit-web-components-dev'",
-    "release:prod-temporary-disable": "echo 'disabling for now node ../../scripts/release/release-prod.js web-components reapit-web-components'",
-    "start:dev": "start-storybook -p 8080",
-    "test:ci-temporary-disable": "echo 'disabling for now cross-env TZ=UTC jest --ci --colors --coverage --silent --forceExit'",
-    "test:dev": "cross-env TZ=UTC jest --watch --verbose",
-    "test:update-badges-temporary-disable": "echo 'disabling for now yarn test:ci && jest-coverage-badges --input src/tests/coverage/coverage-summary.json --output src/tests/badges'"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "dependencies": {
-    "little-loader": "^0.2.0",
-    "lodash.merge": "^4.6.2",
-    "react-icons": "^3.7.0",
-    "styled-components": "^4.4.0"
-  },
+  "private": true,
   "devDependencies": {
-    "@types/little-loader": "^0.2.0",
-    "@types/lodash.merge": "^4.6.6"
+    "@reapit/foundations-ts-definitions": "2020-02-13",
+    "@rollup/plugin-commonjs": "^11.0.2",
+    "@rollup/plugin-node-resolve": "^7.1.1",
+    "@rollup/plugin-replace": "^2.3.1",
+    "@testing-library/svelte": "^1.11.0",
+    "@wessberg/rollup-plugin-ts": "^1.2.17",
+    "rollup": "^1.32.0",
+    "rollup-plugin-babel": "^4.3.3",
+    "rollup-plugin-livereload": "^1.0.4",
+    "rollup-plugin-svelte": "^5.1.1",
+    "rollup-plugin-terser": "^5.2.0",
+    "serverless-offline": "^5.12.1",
+    "serverless-webpack": "^5.3.1",
+    "sirv-cli": "^0.4.5",
+    "svelte-jester": "^1.0.4",
+    "svelte-preprocess": "^3.4.0"
   },
-  "peerDependencies": {
-    "react": "^16.10.2",
-    "react-dom": "^16.10.2"
+  "dependencies": {
```

Comment:
you should include `@reapit/cognito-auth` as a dependency here, so yarn build can build in correct order, and also if we want to use a fixed version of it, just provide the version number in here.

Reply:
another cool thing about this is : hen you make some changes in `@reapit/cognito-auth`, all the tests in all packages which have `@reapit/cognito-auth` as a dependency will also be run by `lerna run test:update-badges`. This make sure our changes in `@reapit/cognito-auth` doesn't break another apps.
The same thing's applied for `@reapit/elements`, which forces you to update snapshot of all apps after making changes in `@reapit/elements`. That's very useful :)

Reply:
Cognito Auth is not a dependency of this package and won't be - this is client credentials auth flow so the client side app does not need the user to login